docker-compose.yml

```diff
@@ -0,0 +1,57 @@
+version: '3.8'
+services:
+  bulwark:
+    image: skewled/bulwark:6.0.0
+    container_name: bulwark
+    depends_on:
+      - bulwark-db
+    networks:
+      static-network:
+        ipv4_address: 172.16.16.2
+    ports:
+      - '5000:5000'
+    expose:
+      - '5000'
+    stop_grace_period: 1m
+    volumes:
+      - bulwark-temp:/bulwark/src/temp:rw
+    command: >
+      sh -c "
+      until mysql --host=172.16.16.3 --user=root --password=bulwark --database=mysql -e 'SELECT user FROM user;'; do
```

Comment:
@skewled Should this command be using env variables to connect to the mysql database rather than hard coded user and password?

Reply:
I'm not sure that works but worth a shot, and if it does, I'll be sure the PR is updated with these set as parameters.